docs/flourishes.md

```diff
@@ -0,0 +1,95 @@
+# Flourish Docs
+
+A flourish is a design patten used for creating ambient background elements to liven up pages and page components. Flourishes are a group of SVG elements (hatching patterns and polygons) that are controlled using CSS for styling and JavaScript for optional animation.
+
+### Flourish top and tail group quick-start
+
+To use an existing flourish group use the `flourish` partial inside the `content` partial block using either the `top` or `tail` postfix ie:
+
+```handlebars
+{{#content "flourish-top"}}
+  {{> flourish flourish-top }}
+{{/content}}
+```
+
+The argument passed to `flourish` partial is `flourish-top` markdown content (currently in YMAL format) ie:
+
+```yaml
+flourish-top:
+  baseClass: flourish-group-a
+  shapes:
+  - shape: polygon-shape-4
+    shapeClass: polygon-shape-4
+    reveal: true
+    exclude: bottom
+  - shape: hatching-pattern-3
+    shapeClass: hatching-pattern-3
+    reveal: true
+    exclude: bottom
+```
+
+**Note:** To reuse a flourish that has been configured for the opposite end add `flip: true`
+
+### Options
+
+```yaml
+baseClass: {string} - Class name for Flourish group to control group style
+shape: {string} - The SVG shape to apply
+shapeClass: {string} - Class name to control shape style
+
+# Optional
```

Comment:
Where should I use these, at a flourish level or at a shape level?

Reply:
Changed to reflect this